app/assets/javascripts/graphs/segment_history.js

```diff
@@ -0,0 +1,76 @@
+$(document).on('runs#show:loaded', function() {
+  if(gon.run.program !== "livesplit") {
+    return
+  }
+
+  color_styles = [
+    "rgba(14, 144, 210, 1)",
+    "rgba(128, 87, 165, 1)",
+    "rgba(94, 185, 94, 1)",
+    "rgba(250, 210, 50, 1)",
+    "rgba(221, 81, 76, 1)",
+    "rgba(243, 123, 29, 1)"
+  ]
```

Comment:
Since every segment has its own graph can all the colors except the one be removed?

Reply:
Ooo yeah, they can!

Reply:
oooo yes it can!